Source/SharpNeedle/SonicTeam/SHLightField.cs

```diff
@@ -0,0 +1,79 @@
+namespace SharpNeedle.SonicTeam;
+
+using System.IO;
+using BINA;
+using Utilities;
+
+[NeedleResource("st/shlf", @"\.(sh?)lf$")]
+public class SHLightField : BinaryResource
+{
+    public uint FormatVersion { get; set; } = 1;
+    public float[] Field04 { get; set; } = new float[36];
```

Comment:
Looking at this made me realize this data is probably default probe lighting data in case an object doesn't fall inside any of the probe volumes. A single probe has 9 colors, 4 components, and 9 * 4 would be 36.

You could rename this to something like DefaultProbeLightingData or something, whatever you think is the best.

Reply:
Solved in b28969f.